src/hooks/useUIInstance.ts

```diff
@@ -32,27 +35,47 @@ export const useUIInstance = (): ConsoleDotEnvironments => {
   return getInternalInstance() ?? env;
 };
 
-const SBOM_PLACEHOLDER = '<PLACEHOLDER>';
-const getSBOMEnvUrl = (env: ConsoleDotEnvironments) => (imageHash: string) => {
-  if (env === ConsoleDotEnvironments.prod) {
-    return `https://atlas.devshift.net/sbom/content/${SBOM_PLACEHOLDER}`.replace(
-      SBOM_PLACEHOLDER,
-      imageHash,
-    );
-  }
-  return `https://atlas.stage.devshift.net/sbom/content/${SBOM_PLACEHOLDER}`.replace(
-    SBOM_PLACEHOLDER,
-    imageHash,
+const getBombinoUrl = (
+  notifications: { title: string; event: string; config?: { url: string } }[],
+) => {
+  const notification = notifications.find(
+    (n) => n.title === 'SBOM-event-to-Bombino' && n.event === 'repo_push',
```

Comment:
nit/question: might be a silly question, but is `'SBOM-event-to-Bombino'` a constant or documented somewhere? just making sure it's safe to hardcode here 🤔